tests/Unit/NumericalAlgorithms/Spectral/Test_SwshTestHelpers.cpp

```diff
@@ -0,0 +1,62 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cmath>
+#include <complex>
+#include <random>
+
+#include "Utilities/Math.hpp"
+#include "tests/Unit/NumericalAlgorithms/Spectral/SwshTestHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+#include "tests/Utilities/MakeWithRandomValues.hpp"
+
+namespace Spectral {
+namespace Swsh {
+namespace TestHelpers {
+SPECTRE_TEST_CASE("Unit.NumericalAlgorithms.Spectral.SwshTestHelpers",
+                  "[Unit][NumericalAlgorithms]") {
+  CHECK(factorial(5) == 5.0 * 4.0 * 3.0 * 2.0);
```

Comment:
why not just compare a few to the one in `ConstantExpressions` for these small numbers?